models/customer.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from datetime import datetime
+
+class Customer:
+    def __init__(self, url="http://localhost:5000"):
+        self.url = url
+
+    def add_customer(self, name=None, postal_code=None, phone=None):
+        query_params = {
+            "name": name,
+            "postal_code": postal_code,
+            "phone": phone
+        }
+        response = requests.post(self.url+"/customers",json=query_params)
+        return response.json()
+
+    def update_customer(self, customer_id=None, name=None, postal_code=None,
+        phone=0):
+
+        query_params = {
+            "name": name,
+            "postal_code": postal_code,
+            "phone": phone
+        }
+        response = requests.put(self.url+f"/customers/{customer_id}",
+            json=query_params)
+        if response.status_code != 200:
+            return "Could not find customer by that id"
+        print("response:", response)
```

Comment:
You should also leave interacting with the user (print and input) to `main.py` so each module has separate roles. 

Of course these could also be debug statements you left in.